File: ejercicios/python/ejercicio045.py
n = int(input("Ingrese un nœmero entero: "))
tienedivisior = False

if (n >= 2):
    tienedivisior = False
    
    if (n >= 2):
        tienedivisor = False
        i = 2;
        while (i < n):
            if (n % i == 0):
                tienedivisor = True
                i = n
            i = i + 1
        
        if (not tienedivisor):
            print(str(n) + " es un nœmero primo")
        else:
            print(str(n) + " no es un nœmero primo")
    else:
        print(str(n) + " debe ser un nœmero mayor o igual a 2")

File: ejercicios/c++/ejercicio045.cpp
#include<iostream>

using namespace std;

int main() {
	int i, n;
	bool tienedivisor=false;
	
	cout<<"Ingrese un nœmero entero: ";
	cin>>n;
	
	if (n>=2) {
		tienedivisor=false;
		i=2;
		while (i<n) {
			if (n%i==0) {
				tienedivisor=true;
				i=n;
			}
			i=i+1;
		}
		if (!tienedivisor) {
			cout<<n<<" es un nœmero primo"<<endl;
		} else {
			cout<<n<<" no es un nœmero primo"<<endl;
		}
	} else {
		cout<<n<<" debe ser un nœmero mayor o igual a 2"<<endl;
	}
return 0;
}

